<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
<title>Lizhong Chen (陈力仲) </title>
</head>
<body>
<div id="layout-content">
<div id="toptitle">
<h1>Lizhong Chen (陈力仲) </h1>
</div>
<table class="imgtable"><tr><td>
<img src="visa.jpg" alt="alt text" width="150px" height="150px" />&nbsp;</td>
<td align="left"><p>Lizhong Chen <br />
Research Officer <br /> 
<a href="https://www.wehi.edu.au/research/research-fields/bioinformatics">Bioinformatics Division</a> <br />
<a href="https://www.wehi.edu.au/">Walter and Eliza Hall Institute of Medical Research (WEHI)</a>  <br />
Contact: <a href="mailto:chen.l@wehi.edu.au">Email</a>, <a href="https://scholar.google.com.au/citations?user=dqb0ajUAAAAJ&amp;hl=en">Google Scholar</a>, <a href="https://www.linkedin.com/in/lizhongc">Linkedin</a>, <a href="https://github.com/lizhongc">Github</a>, <a href="Academic_CV.pdf">CV</a></p>
</td></tr></table>
<h2>About me</h2>
<p>I am a research officer in <a href="https://www.wehi.edu.au/people/gordon-smyth">Prof. Gordon K Smyth</a> lab team at Bioinformatics Division, Walter and Eliza Hall Institute of Medical Research (WEHI). I completed my PhD degree in Statistics from the University of Melbourne under the supervision of <a href="https://findanexpert.unimelb.edu.au/profile/144561-guoqi-qian">A/Prof. Guoqi Qian</a>, Prof. Yuriy Kuleshov and <a href="https://findanexpert.unimelb.edu.au/profile/795463-tingjin-chu">Dr. Tingjin Chu</a> in Jul. 2022. I received my M.S. and B.S. in Mathematics from Peking University on Jun. 2016 and Jul. 2013, respectively.</p>
<h2>Education </h2>
<ul>
<li><p>Ph.D. in Statistics, School of Mathematics and Statistics, The University of Melbourne, Victoria, Australia, Feb. 2017 - Feb. 2022</p>
</li>
<li><p>M.S. in Mathematics, School of Mathematical Science, Peking University, Beijing, China, Sep. 2013 - Jun. 2016</p>
</li>
<li><p>B.S. in Mathematics, Yuanpei College, Peking University, Beijing, China, Sep. 2009 - Jul. 2013</p>
</li>
</ul>
<h2>Research Interests </h2>
<ul>
<li><p>Generalized linear models  </p>
</li>
<li><p>Bioinformatics: limma, edgeR</p>
</li>
</ul>
<h2>Publications </h2>
<ul>
<li><p>Journal articles</p>
<ul>
<li><p>Chen Y, <b>Chen L</b>, Lun AT, Baldoni PL and Smyth GK. edgeR v4: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets.  <i>Nucleic Acids Research,</i> 53(2), 2025. <a href="https://doi.org/10.1093/nar/gkaf018">doi</a><br /></p>
</li>
<li><p>Baldoni PL, <b>Chen L</b>, Smyth GK. Faster and more accurate assessment of differential transcript expression with Gibbs sampling and edgeR v4, <i>NAR Genomics and Bioinformatics,</i> 6(4), 2024. <a href="https://doi.org/10.1093/nargab/lqae151">doi</a> </p>
</li>
<li><p>Qian G, <b>Chen L</b> and Kuleshov Y. Improving Methodology for Tropical Cyclone Seasonal Forecasting in the Australian and the South Pacific Ocean Regions by Selecting and Averaging Models via Metropolis&ndash;Gibbs Sampling, <i>Remote Sensing,</i> 14(22), 5872. <a href="https://doi.org/10.3390/rs14225872">doi</a> <br /></p>
</li></ul>
</li>
<li><p>Preprint</p>
<ul>
<li><p>Baldoni PL, <b>Chen L</b>, Smyth GK. Faster and more accurate assessment of differential transcript expression with Gibbs sampling and edgeR 4.0. <i>bioRxiv,</i> 2024. <a href="https://doi.org/10.1101/2024.06.25.600555">doi</a> <br /></p>
</li>
<li><p>Wang J, <b>Chen L</b>, Thijssen R, Phipson B, Speed TP. GLMsim: a GLM-based single cell RNA-seq simulator incorporating batch and biological effects. <i>bioRxiv,</i> 2024. <a href="https://doi.org/10.1101/2024.03.20.586030">doi</a> <br /></p>
</li>
<li><p>Chen Y, <b>Chen L</b>, Lun AT, Baldoni PL and Smyth GK. edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets. <i>bioRxiv,</i> 2024. <a href="https://doi.org/10.1101/2024.01.21.576131">doi</a><br /></p>
</li></ul>
</li>
<li><p>Ph.D. Thesis</p>
<ul>
<li><p>Model selection and averaging by Gibbs sampler with a tropical cyclone seasonal forecasting application, Feb. 2022. <a href="http://hdl.handle.net/11343/311691">Minerva Access</a></p>
</li>
</ul>

</li>
</ul>
<h2>Presentations </h2>
<ul>
<li><p>edgeR quasi-likelihood: correcting deviance for bias, 2022, WEHI Bioinformatics Divsion seminar</p>
</li>
<li><p>Extending edgeR for small counts and large samples, 2023, WEHI Bioinformatics Divsion seminar</p>
</li>
<li><p>Applying edgeR for alternative splicing analysis, 2024, WEHI Bioinformatics Divsion seminar</p>
</li>
</ul>
<h2>Softwares</h2>
<ul>
<li><p>R package: edgeR (Empirical Analysis of Digital Gene Expression Data in R) <br />
We develop the new quasi-likelihood method for differential expression analysis using the exact calculation for unit deviance of Negative Binomial distribution. It is available on <a href="https://bioconductor.org/packages/release/bioc/html/edgeR.html">R-Bioconductor</a>.</p>
</li>
<li><p>R package: statmod (statmod: Statistical Modeling) <br /> 
We make the exact distributional calculations for unit deviances of Binomial, Poisson and Negative Binomial distributions. It is available on <a href="https://cran.r-project.org/web/packages/statmod/index.html">R-CRAN</a>.</p>
</li>
<li><p>R package: IBGS (Iterated Blockwise Gibbs Sampler) <br />
IBGS is an MCMC search algorithm designed to find the best model in the high dimensional data given a model selection criterion such as AIC, BIC and so on. Moreover, it can identify those most important covariates having the significant influence on the response. It is available on <a href="https://github.com/lizhongc/IBGS">Github</a>.</p>
</li>
</ul>
<div id="footer">
<div id="footer-text">
Page generated 2025-01-27 11:47:27 AEDT, by <a href="http://jemdoc.jaboc.net/">jemdoc</a>.
</div>
</div>
</div>
</body>
</html>
